sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerConfig.java

```diff
@@ -43,7 +43,7 @@ public abstract class SpannerConfig implements Serializable {
   // A default host name for batch traffic.
   private static final String DEFAULT_HOST = "https://batch-spanner.googleapis.com/";
   // Deadline for Commit API call.
-  private static final Duration DEFAULT_COMMIT_DEADLINE = Duration.standardSeconds(15);
+  private static final Duration DEFAULT_COMMIT_DEADLINE = Duration.standardSeconds(60);
```

Comment:
I actually think we can override the commit deadline specifically only for metadata table writes here: https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/MetadataSpannerConfigFactory.java#L87

We probably don't want to override the deadline for all Spanner commits.

Reply:
Thank you for pointing this out! Made the change in a new commit.